types/fetch-mock/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "dtslint/dt.json" }
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "strict-export-declare-modifiers": false
```

Comment:
Well, I could wrap everything in a `declare module` to avoid all of the `strict-export-declare-modifiers` issues. I’m not sure whether this is [best practice](http://definitelytyped.org/guides/best-practices.html). Lots of other packages do it though. According to [this comment](https://github.com/Microsoft/TypeScript-Handbook/issues/180#issuecomment-195452691) by @mhegazy this changes it from an “ambient module” (or just “module”?) to an "ambient module declaration”. But does that change anything in practice? It’s not polluting the global namespace in any of the two variants, is it? Maybe @andy-ms can advise?